app/views/handlers/feedback.py

```diff
@@ -58,7 +62,28 @@ def get_page_title(self):
         return self.PAGE_TITLE
 
     def handle_post(self):
-        self._session_store.session_data.feedback_count += 1
+        session_data = self._session_store.session_data
+        session_data.feedback_count += 1
+
+        feedback_metadata = FeedbackMetadata(
+            session_data.feedback_count,
+            self._schema.form_type,
+            session_data.language_code,
+            self._schema.region_code,
+            session_data.tx_id,
+        )
+
+        feedback_message = FeedbackPayload(
+            self.form.data.get("feedback_text"),
+            self.form.data.get("feedback_type"),
+            self.form.data.get("feedback_type_question_category"),
+        )
```

Comment:
The ids are wrong. They should be:
```suggestion
        feedback_message = FeedbackPayload(
            self.form.data.get("feedback-text"),
            self.form.data.get("feedback-type"),
            self.form.data.get("feedback-type-question-category"),
        )
```